data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
-			`	Two different Elders respond simultaneously. The first says, "Five thousand Hai in two hundred years is barely a concern. That many managed to get themselves killed swimming with Gelo Appanoh in the last decade alone."`
-			`	That one is left searching the room for someone who might actually be listening halfway through his sentence, though, as the second points at Teeneep and loudly says, "You wouldn't dare!" To which Teeneep responds, "It is already scheduled to happen unless I actively stop it!"`
-			`	Sayari has to raise her voice above the din and disentangles their two responses for you as the room devolves into shouting and several Elders who look quite worried leave the room. A moment later all three of you are escorted out and the chambers are closed.`
+			`Upon your return, Teeneep lodges a formal demand for another audience with the Elders, including the context that she has acquired support from the Unfettered with appropriate proof. Once sent, Sayari steps out for a while to talk to someone and ensure that the demand is being processed with appropriate speed, leaving you and Teeneep alone in her office.`
+			`	Teeneep spends a few moments more doing some other things at her terminal before finally sitting back with an expression of quiet determination. In the quiet she turns to you from her seat and says, "Well, all the pieces are set, now I suppose all that remains is to play out the moves."`
+			label teeneepquestions
+			choice
+				`	"What will happen now then?"`
+					to display
+						not "Hai Reveal: what will happen now?"
+				`	"Do you think the Elders will listen?"`
+					goto listen
+					to display
+						not "Hai Reveal: will the elders listen?"
+				`	"Will there truly be a war between Hai and Unfettered?"`
+					goto war
+					to display
+						or
+							has "Hai Reveal: what will happen now?"
+							has "Hai Reveal: will the elders listen?"
+						not "Hai Reveal: will there be war?"
+				`	"What are the other options?"`
+					goto options
+					to display
+						has "Hai Reveal: will there be war?"
+				`	"Who is Aimet?"`
+					goto aimet
+					to display
+						has "Hai Reveal: will there be war?"
+			action
+				set "Hai Reveal: what will happen now?"
+			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into an actual war. Our brethren are no longer merely a disaffected rabble of veterans camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`
```

Comment:
```suggestion
			`	She shrugs. "It is honestly quite hard to say. I struggle to believe that the Elders could be so unwilling to act that they would risk escalating the standoff with the Unfettered into a real war. Our brethren are no longer merely a disaffected rabble of veterans, camping on the one world that supported radical change. They are a truly empowered movement with consolidated support at home and sympathizers amongst the Hai at large."`
```
I don't know if you agree with this change, but I like to think that their war has been fake all along, even though the fighting was real.